doc/examples/segmentation/plot_chan_vese.py

```diff
@@ -0,0 +1,84 @@
+"""
+======================
+Chan-Vese Segmentation
+======================
+
+The Chan-Vese Algorithm is an algorithm designed to segment objects
+without clearly defined boundaries. This algorithm is based on level
+sets and that are evolved iteratively to minimize an energy, which is
+defined by weighted values corresponding to the sum of differences in
+intensity from the average value outside the segmented region, the sum
+of differences from the average value inside the segmented region, and
+a term which is dependent on the length of the boundary of the
+segmented region.
+
+This algorithm was first proposed by Tony Chan and Luminita Vese, in
+an publicaion entitled "An Active Countour Model Without Edges" [1]_.
+
+This implementation of the algorithm is somewhat simplified.
+The arguments are as follows:
+
+lambda1 : weight for the difference from the average 'inside' the
```

Comment:
I don't think we want the parameters listed here. They are already in the function signature. Please, remove here.